modules/nw-using-cookies-keep-route-statefulness.adoc

```diff
@@ -19,3 +19,10 @@ for the session. The cookie is passed back in the response to the request and
 the user sends the cookie back with the next request in the session. The cookie
 tells the Ingress Controller which endpoint is handling the session, ensuring
 that client requests use the cookie so that they are routed to the same pod.
+
+[NOTE]
+====
+Cookies cannot be set on passthrough routes, because the HTTP traffic cannot be seen. Instead, a number is calculated based on the source IP address, which determines the back-end.
+
+If backends change, the traffic can be directed to the wrong server, making it less sticky. If you are using a load balancer, which hides source IP, the same number is set for all connections and traffic is sent to the same pod.
```

Comment:
"Backend" here should match whatever you choose above. If you choose to use "backend" then leave as is.